adr/adr-process.md

```diff
@@ -0,0 +1,26 @@
+# ADR Process
+
+## Overview
+
+In the BigBone project, we use ADRs to log the outcome of design and decision making activities. Hence, we can view them
```

Comment:
We should write down in full what “ADR” means at least once here at the start of the document. At least I didn’t know that abbreviation.

Reply:
Good point. I had it written out somewhere but it seems it did not make it into this PR. Will fix it. 👍